governance/votes/Executive Vote - June 27, 2024.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Template - [Executive Vote] SBE Parameters Update, Launch Project Funding, Spark Proxy Spell - June  27, 2024
+summary: Update Smart Burn Engine (SBE) DssFlapper parameters, transfer funding to the Launch Project, and trigger a Spark proxy spell.
+date: 2024-06-27T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] SBE Parameters Update, Launch Project Funding, Spark Proxy Spell - June 27, 2024
+
+The Governance Facilitators and Dewiz, Sidestream, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
```

Comment:
```suggestion
If this executive proposal passes, the following **actions** will occur within the Maker Protocol:
```
@CivicSage3 I'd use "actions" here because it's more encompassing. The term "changes" doesn't fully capture the scope of activities like fund transfers or triggering the proxy spell, whereas "actions" covers all aspects included in the spell. But I know the other team uses "changes", so it's okay